smf/oximeter/replicated-cluster/manifest.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+
+<service_bundle type='manifest' name='oximeter'>
+
+<service name='oxide/oximeter' type='service' version='1'>
+  <create_default_instance enabled='true' />
+
+  <dependency name='multi_user' grouping='require_all' restart_on='none'
+    type='service'>
+  <service_fmri value='svc:/milestone/multi-user:default' />
+  </dependency>
+
+  <dependency name='zone_network_setup' grouping='require_all' restart_on='none'
+    type='service'>
+  <service_fmri value='svc:/oxide/zone-network-setup:default' />
+  </dependency>
+
+  <method_context>
+    <method_environment>
+      <envvar name="LD_LIBRARY_PATH" value="/opt/ooce/pgsql-13/lib/amd64" />
+    </method_environment>
+  </method_context>
+  <exec_method type='method' name='start'
+    exec='ctrun -l child -o noorphan,regent /opt/oxide/oximeter-collector/bin/oximeter run /var/svc/manifest/site/oximeter/config.toml --address %{config/address} --id %{config/id} --replicated &amp;'
```

Comment:
Just to make sure I understand, this whole file is duplicated so that we can pass the `--replicated` flag to this invocation? If so, is there a way to simplify that at all? E.g., can we provide this as an SMF property or in the TOML configuration file?

Reply:
Hm, let me think about how I can make this nicer

Reply:
I've modified the code so that the `replicated` setting is now a field in the oximeter configuration file instead of a flag in the CLI.

Example:
```toml
[db]
batch_size = 1000
batch_interval = 5 # In seconds
replicated = false

[log]
level = "debug"
mode = "file"
path = "/dev/stdout"
if_exists = "append"
```

We still need to have two configuration files as there really isn't a way to pass some sort of environment variable via [omicron-package](https://github.com/oxidecomputer/omicron-package). But, it's now much more obvious to see what the difference between the files is, and it's cleaner.